x/twap/types/utils.go

```diff
@@ -75,3 +76,13 @@ func SpotPriceTimesDuration(sp sdk.Dec, timeDelta time.Duration) sdk.Dec {
 func AccumDiffDivDuration(accumDiff sdk.Dec, timeDelta time.Duration) sdk.Dec {
 	return accumDiff.QuoInt64(int64(timeDelta))
 }
+
+func LexicographicalOrderDenoms(denom0, denom1 string) (string, string, error) {
```

Comment:
Nice! This totally makes sense. Please consider covering with a test

Reply:
Done!